RevenueCatUI/CustomerCenter/Views/ManageSubscriptionsView.swift

```diff
@@ -303,7 +303,12 @@ struct ManageSubscriptionButton: View {
                                  product: promotionalOfferData.product,
                                  promoOfferDetails: promotionalOfferData.promoOfferDetails)
         })
-        .buttonStyle(ManageSubscriptionsButtonStyle())
+        .applyIf(self.path.type != .cancel) {
+            $0.buttonStyle(ManageSubscriptionsButtonStyle())
+        }
+        .applyIf(self.path.type == .cancel) {
+            $0.foregroundColor(Color.red)
```

Comment:
made it always red because I think it should have good contrast in light and dark modes. But maybe this should be configurable?

Reply:
I think if we allow for configuration of the background colors we should probably allow for configuration of this one because it might have bad contrast with your choice of a background

Reply:
Hmm, this could look weird if the cancel button is not the last button in the paths configured in the backend right?

Reply:
Good point, we can make it so it is always the last one? We can guarantee that from the frontend maybe adding the cancel button as a checkbox instead of part of the configurable buttons. Or we could also leave it as part of the devs responsibilities. 

Let's talk about this in the sync, because I am noticing now this might make it more complex than we want at least for the first iterations